src/components/CompanyCard/index.js

```diff
@@ -163,22 +168,20 @@ const CompanyCard = ({
                 : lng.moreInfo.no}
               <Tooltip
                 text={lng.moreInfo.coronavirusCancellationCover.tooltip}
-                side="left"
+                side={{ xs: 'bottom', md: 'left' }}
                 minWidth="260px"
                 margin={{ left: '5px' }}
-              />
+              >
+                <TooltipAnchor>{lng.moreInfo.info}</TooltipAnchor>
+              </Tooltip>
             </Info>
           )}
           {/** Medical Equipment Cover */}
           {medical_equipment_cover && (
             <Info>
               <InfoTitle>{`${lng.moreInfo.medicalEquipmentCover.title} - `}</InfoTitle>
               {medical_equipment_cover.offers_cover
-                ? `${
-                    lng.moreInfo.medicalEquipmentCover.offered
-                  }${medical_equipment_cover.cover_amount
-                    .toString()
-                    .replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',')}`
+                ? `${lng.moreInfo.medicalEquipmentCover.offered} ${medical_equipment_cover.cover_amount}`
```

Comment:
Could we remove the space here or put the `£` sign into the `cover_amount` text in order to keep the currency and amount together? 

![image](https://user-images.githubusercontent.com/6080548/94687671-7adde300-0324-11eb-907a-8f702b3fb266.png)